### PR TITLE
Fix decimals

### PR DESCRIPTION
**Pull request by toshiakitakase**

### CODE REVIEW

File: frontend/packages/react-app/src/components/molecules/TokenBalance/index.tsx [hunk 1]
```diff
@@ -1,20 +1,29 @@
...
   <>
     <Box display="flex" style={{ alignItems: "baseline" }}>
       <Box mr={4}>
         <Typography>{itemName}</Typography>
       </Box>
       <Box mr={1}>
-        <Typography variant={"h4"}>{balance}</Typography>
+        <Typography variant={"h4"}>
+          {getBalanceDevidedByDecimals(balance, decimals)}
+        </Typography>
```
Comment (wild-mouse) [inline frontend/packages/react-app/src/components/molecules/TokenBalance/index.tsx:26]: We have a `TokenAmount` component and you can use it. Is it not suitable? 
https://github.com/iroiro/iroiro/tree/develop/frontend/packages/react-app/src/components/atoms/TokenAmount

Comment (toshiakitakase) [reply frontend/packages/react-app/src/components/molecules/TokenBalance/index.tsx]: Fixed

**Comment by wildmouse** [comment]
Sorry for late review, but Error on storybook for `TokenBalance` component: 

```
[DecimalError] Invalid argument: 8888888e-undefined
```

**Comment by toshiakitakase** [comment]
Fixed!

**Review by wildmouse — APPROVED** [review]
LGTM 🏃

**Pull request merged**